### DISCUSSION TITLE
How do I do the new LTMeat

### DISCUSSION THREAD

**Discussion by Vhirttuegaming** [3kh0/ext-remover#716] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I don't understand what extension page they are talking about so can someone explain how to do the new LTMeat?

**Comment by Vhirttuegaming** [comment]
Can you give me a video if possible?

**Reply by 3kh0** [reply]
It says it in the README with so much detail it is not hard to figure it out. Come on, please use your brain

**Reply by Vhirttuegaming** [reply]
well what extenstion page and am I going to do it a bunch of times?

**Reply by AshtonDavies** [reply, marked as answer]
> well what extenstion page and am I going to do it a bunch of times?

Any. You would typically use either manifest.json or _generated_background_page.html it's really not that complicated unless your brain went through the dryer or something.

**Reply by Vhirttuegaming** [reply]
so basically make a bookmark with 800+ extensions then open it

**Reply by Vhirttuegaming** [reply]
3kh0 do you have discord? We can talk there.

**Reply by AshtonDavies** [reply]
> so basically make a bookmark with 800+ extensions then open it

Open and close, then open a second time, then flip the switch. 😵‍💫

**Reply by Vhirttuegaming** [reply]
I'm a very visual person so can you just give me a video for all the steps. I understand you guys are calling me dumb but I'm not really a tech savvy person.

**Reply by AshtonDavies** [reply]
https://github.com/3kh0/ext-remover/discussions/671#discussioncomment-6770771
And make sure to click the link in that comment if that's not common sense.

**Reply by Vhirttuegaming** [reply]
can you make video showing me how to make the hack not doing it?
